src/NJsonSchema.CodeGeneration.TypeScript/Templates/ConvertToClass.liquid

```diff
@@ -11,8 +11,10 @@ if (Array.isArray({{ Value }})) {
 {%   if IsArrayItemNewableObject -%}
         {{ Variable }}{% if RequiresStrictPropertyInitialization %}!{% endif %}.push({{ ArrayItemType }}.fromJS(item{% if HandleReferences %}, _mappings{% endif %}));
 {%   else -%}
-{%     if IsArrayItemDate or IsArrayItemDateTime -%}
+{%     if IsArrayItemDate -%}
         {{ Variable }}{% if RequiresStrictPropertyInitialization %}!{% endif %}.push({{ StringToDateCode }}(item));
```

Comment:
Should be StringToDateOnlyCode and below StringToDateCode, no?

Reply:
yes, you're right. sorry for mistake

Reply:
resolved